setup.py

```diff
@@ -25,7 +25,8 @@
 # Dependencies
 extras_require = {
     'serial':   ['pyserial~=3.0'],
-    'neovi':    ['python-ics>=2.12']
+    'neovi':    ['python-ics>=2.12'],
+    'usb-can-analyzer': ['crccheck>=0.6']
```

Comment:
Also requires pyserial.

Reply:
in the last update I intended to put:
'crccheck': ['crccheck>=0.6']
I'll fix that shortly.  Regrading pyserial,  I believe that is already done?  if somethings missing let me know.